ext/date/php_date.c

```diff
@@ -2543,6 +2540,17 @@ PHPAPI bool php_date_initialize_from_ts_double(php_date_obj *dateobj, double ts)
 	usec = (int)(fmod(ts, 1) * 1000000);
 
 	if (UNEXPECTED(usec < 0)) {
+		if (UNEXPECTED(sec == TIMELIB_LONG_MIN)) {
+			zend_throw_error(
+				date_ce_date_range_error,
+				"Seconds must be a finite number between " TIMELIB_LONG_FMT " and " TIMELIB_LONG_FMT ", %g given",
+				TIMELIB_LONG_MIN,
+				TIMELIB_LONG_MAX,
+				sec_dval
```

Comment:
Not printing usec would be confusing here.

Reply:
Mh to be precise the timestamp must be between TIMELIB_LONG_MIN.0 and TIMELIB_LONG_MAX.999999.
Should this be included in the message as well as currently it's about the seconds part only.

Reply:
I have updated the message to `Seconds must be a finite number between {TIMELIB_LONG_MIN} and {TIMELIB_LONG_MAX}.999999, {ARGUMENT} given`